docs/modules/ROOT/pages/how-to-guides/proc_maintaining_operator_graphs.adoc

```diff
@@ -0,0 +1,222 @@
+:_content-type: PROCEDURE
+:myfunctionone: maintaining_operator_graphs
+
+[id="maintaining_operator_graphs_{context}"]
+= Maintaining an operator graph
+File-based catalogs (FBC) provide a streamlined approach to store and distribute metadata about operators written for the Operator Lifecycle Manager (OLM). You can use {ProductName} to build, test, and release multiple components from a single FBC repo. 
```

Comment:
Maybe I'm looking at this wrong, but I don't see this paragraph displayed when viewing the preview link. Underneath the title is "FBCs represent...", so I don't see the acronym spelled out before it gets used.